spark3/src/main/java/org/apache/iceberg/spark/procedures/BaseProcedure.java

```diff
@@ -48,20 +46,18 @@ protected BaseProcedure(TableCatalog tableCatalog) {
     this.tableCatalog = tableCatalog;
   }
 
-  protected <T> T modifyIcebergTable(String namespace, String tableName, Function<org.apache.iceberg.Table, T> func) {
-    return execute(namespace, tableName, true, func);
+  protected <T> T modifyIcebergTable(String identifier, Function<org.apache.iceberg.Table, T> func) {
```

Comment:
nit: shall we call it `identifierAsString` to be consistent?

Reply:
changed